webservices/rest.py

```diff
@@ -219,6 +219,16 @@ def add_aggregate_resource(api, view, schedule, label):
     '/committee/<committee_id>/filings/',
     '/candidate/<candidate_id>/filings/',
 )
+
+api.add_resource(
+    filings.EFilingSummaryView,
+    '/efile/presidential/<string:form_type>/',
```

Comment:
No that's all right, I based these after this: 

https://18f.slack.com/archives/fec-partners/p1470078912000519

Which this brings me to a question that @noahmanger had.  We also want to add an endpoint that serves up resources similarly too `/reports/<committee-type`, so that they can be viewed together at the efile endpoint.

This was his comment:
  "Just checking: will this be able to be used in tandem with your work on the /reports/ endpoint in the same way  that /filings/ and /efile/filings are similar? Ideally, /efile/presidential/ should return similar responses to /reports/presidential/ (as far as I understand)"

Maybe we should discuss at the check-in what endpoints are expected, and their specific nomenclature?  I'm sure most the work has been done, just want to make sure I'm not missing something.